src/pages/home/report/ReportActionItem.js

```diff
@@ -243,6 +251,14 @@ class ReportActionItem extends Component {
                         />
                     </View>
                 )}
+                {shouldDisplayThreadReplies && (
+                    <ReportActionItemThread
+                        childReportID={`${lodashGet(this.props.action, 'childReportID')}`}
+                        numberOfReplies={lodashGet(this.props.action, 'childVisibleActionCount', 0)}
+                        mostRecentReply={`${lodashGet(this.props.action, 'childLastVisibleActionCreated')}`}
```

Comment:
Unnecessary lodash for single level value lookup with no default
Also, no reason to have string literal here, childLastVisibleActionCreated is a string